docs/docs.trychroma.com/components/header/cloud-signup.tsx

```diff
@@ -68,7 +68,7 @@ const CloudSignUp: React.FC = () => {
           </div>
           <div className="h-full flex items-start justify-end flex-shrink-0 ">
             <Image
-              src="/cloud-art.svg"
```

Comment:
Should we remove the svg file as well to avoid confusion?

Reply:
I think we should keep it, just in case we want to generate the same art in new sizes. Much easier than chasing down a Figma file. WDYT?